x-pack/plugins/graph/public/application.ts

```diff
@@ -12,6 +12,8 @@ import { i18nDirective, i18nFilter, I18nProvider } from '@kbn/i18n/angular';
 import '../../../../webpackShims/ace';
 // required for i18nIdDirective
 import 'angular-sanitize';
+// required for ngRoute
+import 'angular-route';
```

Comment:
This was present previously as a side effect of being importing the legacy dependencies from core 🙈 ?

Reply:
Yeah this scared me at first, but I realized it was a side effect of the legacy import in saved_objects that I removed: https://github.com/elastic/kibana/pull/61011/files#diff-2df2b2c6ecb5d90a9a0d7c0a81023475L36

It ends up importing from `ui/routes` which imports `angular-route`.